docs/index.rst

```diff
@@ -6,6 +6,7 @@ Here you will find all of the provider documentation...
 .. toctree::
    :caption: Public APIs:
 
+   executors
```

Comment:
I'm not sure what this line is about.

Reply:
This includes the executors module documentation to the docs.